requirements.txt

```diff
@@ -1,5 +1,4 @@
-# featuretools>=1.0.0
-featuretools @ git+https://github.com/alteryx/featuretools.git@woodwork-integration
+featuretools>=1.0.0
```

Comment:
Good point. I'll check

Reply:
It fails due to woodwork serialization refactor.

Reply:
That works

Reply:
Should we consider ft 1.0 minimum fine then?  By default it will install latest featuretools and woodwork, which is fine.  Older 1.x versions of featuretools work when a compatible woodwork version is used.

@thehomebrewnerd

Reply:
Yeah, seems fine to me to leave min at `1.0.0` since there does exist a combination of libraries that will run this successfully.